src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.js

```diff
@@ -36,18 +40,29 @@ export const MappingProfilesTransformationsModal = ({
   onCancel,
 }) => {
   const [isFilterPaneOpen, setFilterPaneOpen] = useState(true);
+  const [searchValue, setSearchValue] = useState(initialSearchFormValues.searchValue);
   const [searchFilters, setSearchFilters] = useState(initialSearchFormValues.filters);
 
   const totalSelectedCount = 0;
 
+  const resetSearchForm = useCallback(() => {
+    setSearchFilters(initialSearchFormValues.filters);
+    setSearchValue(initialSearchFormValues.searchValue);
+  }, []);
+
   useEffect(() => {
     if (!isOpen) {
       setFilterPaneOpen(true);
-      setSearchFilters(initialSearchFormValues.filters);
+      resetSearchForm();
     }
-  }, [isOpen]);
+  }, [isOpen, resetSearchForm]);
 
-  const searchResults = initialTransformationsValues.transformations
+  const searchValueResults = !searchValue
```

Comment:
It looks cool.

Reply:
Agree, I knew that you would pay attention to this :)